src/main/java/com/example/java/test/junior/developer/security/SecurityConfig.java

```diff
@@ -0,0 +1,51 @@
+package com.example.java.test.junior.developer.security;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityCustomizer;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
+import org.springframework.security.web.util.matcher.RequestMatcher;
+
+
+@Configuration
+@EnableMethodSecurity(securedEnabled = true)
+@PropertySource(value = "classpath:application.properties", factory = ApplicationPropertySourceFactory.class)
```

Comment:
Why do you need it? application.properties is default Spring Boot properties file and it fetched by Spring Boot by default. Remove.